src/App.js

```diff
@@ -0,0 +1,33 @@
+import { BrowserRouter,Route,Routes } from 'react-router-dom';
+// import './App.css';
+import Login from './Pages/login';
+import Welcome from './Pages/welcome';
+
+function App() {
+  return (
+    <BrowserRouter >
+      <Routes>
+        <Route path='/' element={<Login />}></Route>
+        <Route path='/welcome' element={<Welcome />}></Route>
+      </Routes>
+    </BrowserRouter>
```

Comment:
Something i feel would be good here, is to separate the routing to a different file. This is fine for now as you only have the 2 routes, but when the routing gets complex and scales it's easy to manage and read if we have separate files for routing. And furthermore sub routing files for individual main routes when expanding further.

For example if the base URL is `example.com/` and we have several role based routes like with System2 like `/a`, `/c` etc., it helps to have separate files for each of these routes and define the sub routes after these in them.

Reply:
Fixed 59ad5293a7296e1c51744aab7f9180a335e0b22b